e2e/src/usecases/RestoreAccountOnboarding.js

```diff
@@ -1,90 +1,90 @@
+      // Set and verify pin
+      await enterPinUi()
+      await enterPinUi()
 
-  // Restore existing wallet
-  it('Restore Wallet Backup', async () => {
-    // wait for connecting banner to go away
-    // TODO measure how long this take
-    await waitFor(element(by.id('connectingToCelo')))
-      .not.toBeVisible()
-      .withTimeout(20000)
+      // wait for connecting banner to go away
+      // TODO measure how long this take
+      await waitFor(element(by.id('connectingToCelo')))
+        .not.toBeVisible()
+        .withTimeout(20000)
 
-    await element(by.id('ImportWalletBackupKeyInputField')).tap()
-    await element(by.id('ImportWalletBackupKeyInputField')).replaceText(`${SAMPLE_BACKUP_KEY}`)
-    if (device.getPlatform() === 'ios') {
-      // On iOS, type one more space to workaround onChangeText not being triggered with replaceText above
-      // and leaving the restore button disabled
-      await element(by.id('ImportWalletBackupKeyInputField')).typeText('\n')
-    } else if (device.getPlatform() === 'android') {
-      // Press back button to close the keyboard
-      await device.pressBack()
-    }
+      // enter the recovery phrase
+      await element(by.id('ImportWalletBackupKeyInputField')).tap()
+      await element(by.id('ImportWalletBackupKeyInputField')).replaceText(phrase)
+      if (device.getPlatform() === 'ios') {
+        // On iOS, type one more space to workaround onChangeText not being triggered with replaceText above
+        // and leaving the restore button disabled
+        await element(by.id('ImportWalletBackupKeyInputField')).typeText('\n')
+      } else if (device.getPlatform() === 'android') {
+        // Press back button to close the keyboard
+        await device.pressBack()
+      }
 
-    await waitFor(element(by.id('ImportWalletButton')))
-      .toBeVisible()
-      .withTimeout(1000 * 5)
-    await element(by.id('ImportWalletButton')).tap()
-  })
+      // start wallet import
+      await waitFor(element(by.id('ImportWalletButton')))
+        .toBeVisible()
+        .withTimeout(1000 * 5)
+      await element(by.id('ImportWalletButton')).tap()
 
-  it('Verify Education', async () => {
-    await waitForElementId('PhoneVerificationSkipHeader')
+      // verification step comes after restoring wallet, skip this step
+      await waitForElementId('PhoneVerificationSkipHeader')
+      await element(by.id('PhoneVerificationSkipHeader')).tap()
+      await element(by.id('PhoneVerificationSkipDialog/PrimaryAction')).tap()
 
-    // skip
-    await element(by.id('PhoneVerificationSkipHeader')).tap()
-    // confirmation popup skip
-    await element(by.id('PhoneVerificationSkipDialog/PrimaryAction')).tap()
-  })
+      // verify that we land on the home screen
+      await expect(element(by.id('SendOrRequestBar'))).toBeVisible()
 
-  it('Wallet Home', async () => {
-    await expect(element(by.id('SendOrRequestBar'))).toBeVisible()
-  })
+      // verify that the correct account was restored
```

Comment:
i updated the assertion to check for the account address rather than checking for wallet home, seems more accurate (although we don't test using this wallet, seeing the correct wallet address restored is a good start)